webosose/ares-cli
# Update version 2.3.0

:Release Notes:
Update version 2.3.0

:Detailed Notes:
- Update version in package.json and npm-shrinkwrap.json
- Update CHANGELOG.md and README.md

:Testing Performed:
1. Install CLI 2.2.0 package using below command
 $ sudo npm uninstall -g @webosose/ares-cli
 $ sudo npm install -g https://github.com:webosose/ares-cli.git#Update_version_2.3.0
2. Check version using below command
 $ ares -V
> Version: 2.3.0

:Issues Addressed:
[WRN-15707] Prepare to release CLI v2.3.0
